frontend/io-webapp/src/presession/QuizList.js

```diff
@@ -5,7 +5,22 @@ import "./quizList.css";
 import "../common.css";
 
 const QuizList = ({ state, dispatch }) => {
-  React.useEffect(() => {
+  const deleteHandler = (quizId) => {
+    (async () => {
+      await fetch(process.env.REACT_APP_BACKEND_URL + "/quiz/" + quizId + "/delete", {
+        method: "POST",
+        headers: {
+          Authorization: state.token,
+          "Content-Type": "application/json",
+        },
+      })
+        .catch((error) => {
+          console.error(error);
+        });
+    })();
+  };
```

Comment:
To przy wywołaniu tej metody chcemy to aktualizować? Widziałem, że jest downloadQuizzes wywoływane zaraz po deleteHandler, jeśli to nie działa to może dlatego, że tu jest asynchronicznie, możliwe, że wywołanie downloadQuizzes bezpośrednio po fetchu wewnątrz async wystarczy, żeby naprawić.

Reply:
Generalnie tak, w tym momencie. Jasne, sprawdzę to rano, dzięki za pomysł, dam znać czy załatwił sprawę :)

Reply:
Dzięki, zadziałało ^^